src/context/online.rs

```diff
@@ -220,4 +220,12 @@ impl AudioContext {
         let opts = node::MediaElementAudioSourceOptions { media_element };
         node::MediaElementAudioSourceNode::new(self, opts)
     }
+
+    /// Returns an [`AudioRenderCapacity`] instance associated with an AudioContext.
+    #[must_use]
+    pub fn render_capacity(&self) -> AudioRenderCapacity {
+        let (sender, receiver) = crossbeam_channel::bounded(1);
+        self.base().add_audio_capacity_listener(sender.clone());
+        AudioRenderCapacity::new(sender, receiver)
```

Comment:
Not sure this is the right thing to do (and somehow related to the previous message), from what I understand from the spec, this should rather be some sort of singleton no?
- 1 call, create and return the instance
- subsequent calls, return a clone() ?